ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -3,7 +3,7 @@ import ZoneId from '../../../../resources/zone_id';
 import { OopsyData } from '../../../../types/data';
 import { OopsyTriggerSet } from '../../../../types/oopsy';
 
-type MitTracker = {
+type EffectTracker = {
```

Comment:
Should this maybe be 'AbiilityTracker`?  Not a strong preference, but `Effect` makes me think of 0x1A lines...